.github/workflows/nmodl-ci.yml

```
@@ -16,7 +16,7 @@ on:
 
 env:
   CTEST_PARALLEL_LEVEL: 1
-  PYTHON_VERSION: 3.8
+  PYTHON_VERSION: 3.9
```

```suggestion
  PYTHON_VERSION: 3.8
```

Forgot to change this back.